src/models/event-timeline.js

```diff
@@ -339,6 +339,27 @@ EventTimeline.prototype.addEvent = function(event, atStart) {
     }
 };
 
+
+/**
+ * Replaces an event in the timeline, if it exists.
+ *
+ * @param {string} eventId  the id of the existing event
```

Comment:
nit: capitalize ID

Reply:
This method will be removed again in the next PR, so should be ok.